docs/apache-airflow/start/docker-compose.yaml

```diff
@@ -23,16 +23,25 @@
 # This configuration supports basic configuration using environment variables or an .env file
 # The following variables are supported:
 #
-# AIRFLOW_IMAGE_NAME         - Docker image name used to run Airflow.
-#                              Default: apache/airflow:|version|
-# AIRFLOW_UID                - User ID in Airflow containers
-#                              Default: 50000
-# AIRFLOW_GID                - Group ID in Airflow containers
-#                              Default: 50000
-# _AIRFLOW_WWW_USER_USERNAME - Username for the administrator account.
-#                              Default: airflow
-# _AIRFLOW_WWW_USER_PASSWORD - Password for the administrator account.
-#                              Default: airflow
+# AIRFLOW_IMAGE_NAME           - Docker image name used to run Airflow.
+#                                Default: apache/airflow:|version|
+# AIRFLOW_UID                  - User ID in Airflow containers
+#                                Default: 50000
+# AIRFLOW_GID                  - Group ID in Airflow containers
+#                                Default: 50000
+#
+# Those configurations are useful mostly in case of standalone testing/running Airflow in test/try-out mode
+#
+# _AIRFLOW_WWW_USER_CREATE     - Whether to create administrator account.
+#                                Default: true
+# _AIRFLOW_WWW_USER_USERNAME   - Username for the administrator account (if requested).
+#                                Default: airflow
+# _AIRFLOW_WWW_USER_PASSWORD   - Password for the administrator account (if requested).
+#                                Default: airflow
+# _AIRFLOW_DB_UPGRADE          - Whether to perform DB upgrade in the init container
```

Comment:
This configuraiton is not customizable by a docker-compose environment variable. It is. a docker image variable. It is hardcoded to true. See: https://github.com/apache/airflow/blob/fb9822222e809aefde68de8aaae4a6d69edd960f/docs/apache-airflow/start/docker-compose.yaml#L152

Reply:
Yep. Removed it (Also create user) which was the same.